lib/cosmos/tools/tlm_grapher/tabbed_plots_tool/tabbed_plots_realtime_thread.rb

```diff
@@ -19,7 +19,7 @@ class TabbedPlotsRealtimeThread < InterfaceThread
 
     # Create a new TabbedPlotsRealtimeThread
     def initialize(tabbed_plots_config, connection_success_callback = nil, connection_failed_callback = nil, connection_lost_callback = nil, fatal_exception_callback = nil)
-      interface = TcpipClientInterface.new('localhost', nil, System.ports['CTS_PREIDENTIFIED'], nil, tabbed_plots_config.cts_timeout, 'PREIDENTIFIED')
+      interface = TcpipClientInterface.new(System.connect_hosts['CTS_PREIDENTIFIED'], nil, System.ports['CTS_PREIDENTIFIED'], nil, tabbed_plots_config.cts_timeout, 'PREIDENTIFIED')
```

Comment:
Its also used above in script.rb and tools.rb (for the api calls to CTS and TLMVIEWER)

This gives the user full control of hosts that sockets listen on.  So if you don't need to ever need to chain servers, you can have the routers only listen on localhost (more secure).  

Or you can do what you said and have the client configured to connect to a server on a different IP (with two different system.txt files).   

Connect and listen have to be differentiated because I can listen to 0.0.0.0 (all ips), but I can't connect to 0.0.0.0.